.travis.yml

```diff
@@ -1,4 +1,3 @@
-sudo: required
```

Comment:
Could we try `sudo: false`? According to @kren1 it's as slow as before.

Reply:
This run is reported to take 38mins, and some other ones are 37mins, so basically the same. But maybe this actually only impacts the startup time not the actual runtime?